main.py

```diff
@@ -141,25 +142,34 @@ def get_inputfiles(args):
 
     return inputfiles
 
+def ats(imagefile, colony, args, config):
+    temperature, end_temperature = auto_temp_schedule(imagefile, colony, args, config)
+    print("auto temperature schedule started")
+    setattr(args, 'start_temp', temperature)
+    setattr(args, 'end_temp', end_temperature)
+    print("auto temperature schedule finished")
+    print("starting temperature is ", args.start_temp, "ending temperature is ", args.end_temp)
+    return args
+
```

Comment:
@jacksujingcheng The `ats` function appears to be a wrapper around `auto_temp_schedule`. I'd recommend just putting it inline in the `main` function; having both `ats` and `auto_temp_schedule` is a little confusing.